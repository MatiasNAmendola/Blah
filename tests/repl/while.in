b: False
while b then
    -1
repeat

c: 8
while c then
    c: c - 1
    c
repeat

True
d: -29
while d + 5 * 4  - (19 - 3) <= 0 then
    d
    if d < 10 and d > -50 then
        d : d - 10
    otherwise
        d : -d
    fi
repeat
d

True
c: 0
while c > -9 then c : c - 1, c repeat
