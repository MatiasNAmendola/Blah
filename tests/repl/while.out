7
6
5
4
3
2
1
0
True
-29
-39
-49
-59
59
True
-1
-2
-3
-4
-5
-6
-7
-8
-9
